common/src/main/java/com/wynntils/screens/base/widgets/TextInputBoxWidget.java

```diff
@@ -115,7 +115,7 @@ public void renderWidget(PoseStack poseStack, int mouseX, int mouseY, float part
 
     protected void doRenderWidget(
             PoseStack poseStack,
-            String renderedText,
+            Pair<String, Integer> renderedTextDetails,
```

Comment:
I have not followed the complete flow, but does this really need to be passed as a pair? As opposed to two individual arguments, that is.

Reply:
It would work perfectly as two individual arguments. I just thought it was more consistent since it is the exact value we receive from `getRenderedHighlighedInterval()`. Please tell me if you think it's better to change

Reply:
Yes, I think it is better to not have a pair. They are mostly useful when you need to return two values at the same time, or possibly when you need to store it in a container of some kind.